microsoft-edge/webview2/release-notes.md

```diff
@@ -19,13 +19,32 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime-specific or SDK-specific.
 
-
 ## Minimum version of the browser or Runtime to load WebView2
 
 To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
 
+## 1.0.992.28
+
+Release Date: September 27, 2021
+
+[NuGet package for WebView2 SDK 1.0.992.28][NuGetGallery1.0.992.28]
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 94.0.992.28 or higher.
+
+### General
+
+#### Bug Fixes
+*   Fixed inconsistent crash in WebView2 process when opening a new window.
+*   Fixed missing WebView2 DLLs (leading to initialization failure) when PlatformTarget isn't set in user's .NET project. \([\#1061][GithubMicrosoftedgeWebviewfeedbackIssue1061]\)
```

Comment:
Is this .NET specific? If it is, should we put it under .NET section?

Reply:
Wondering if its worth having a whole .NET section for just one bullet. I guess in the past we had it because of alot of .NET changes were there, but do you think theres alot of value in having it this time for just one bullet?

Reply:
I'm have no strong opinions either way.